middle_end/flambda2/simplify/flow/mutable_unboxing.ml

```diff
@@ -239,6 +239,13 @@ let blocks_to_unbox ~escaping ~source_info ~required_names =
                         (fun _ -> Flambda_kind.With_subkind.naked_float)
                         fields
                   }
+                | Mixed (tag, shape) ->
+                  let fields_kinds =
+                    List.map Flambda_kind.With_subkind.anything
+                      (Array.to_list
+                         (Flambda_kind.Mixed_block_shape.field_kinds shape))
```

Comment:
This seems to appear quite often. It should maybe be moved to a function in `Flambda_kind.Mixed_block_shape`

Reply:
It looks like it occurs only once (here), which may not count as often enough to bother with an auxiliary function.